tests/draft-next/propertyDependencies.json

```diff
@@ -0,0 +1,102 @@
+[
+    {
+        "description": "propertyDependencies doesn't act on non-objects",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": false}
+            }
+        },
+        "tests": [
+            {
+                "description": "ignores booleans",
+                "data": true,
+                "valid": true
+            },
+            {
+                "description": "ignores integers",
+                "data": 123,
+                "valid": true
+            },
+            {
+                "description": "ignores floats",
+                "data": 1.0,
+                "valid": true
+            },
+            {
+                "description": "ignores string",
+                "data": "abc",
+                "valid": true
+            },
+            {
+                "description": "ignores arrays",
+                "data": [],
+                "valid": true
+            },
+            {
+                "description": "ignores null",
+                "data": null,
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "propertyDependencies doesn't act on non-string property values",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": false}
+            }
+        },
+        "tests": [
+            {
+                "description": "ignores booleans",
+                "data": {"foo": false},
+                "valid": true
+            },
+            {
+                "description": "ignores integers",
+                "data": {"foo": 2},
+                "valid": true
+            },
+            {
+                "description": "ignores floats",
+                "data": {"foo": 1.1},
+                "valid": true
+            },
+            {
+                "description": "ignores objects",
+                "data": {"foo": {}},
```

Comment:
Can you add another one for `{"foo": { "bar": "baz" } }` to make it clear that bar as a property name inside the object is also ignored?

Reply:
What confusion do you expect this to clear up?

Reply:
By reading the first paragraph of the spec and missing (or misinterpreting the rest) - that the value under the property value must be a string.  Also there could be a code error that gets the number of levels wrong between the top object and the terminal value (such as a confusion with `dependentSchemas` perhaps).

Reply:
I added a couple more variations.